resources/views/categoryPage.blade.php

```diff
@@ -99,43 +99,17 @@
 	</div>
 	<!-- END CARD COLUMNNS -->
 </div>
+
 <!-- End Row -->
 
 <!-- Row Fluid -->
 <div class="row-fluid">
-	<!-- PAGINATION -->
-	<nav aria-label="Page navigation">
-		<ul class="pagination justify-content-center">
-			<!-- Previous -->
-			<li class="page-item disabled">
-				<a class="page-link" href="#">
-					<span>&lsaquo;</span>
-				</a>
-			</li>
-			<li class="page-item">
-				<a class="page-link" href="#">1</a>
-			</li>
-			<li class="page-item">
-				<a class="page-link" href="#">2</a>
-			</li>
-			<li class="page-item">
-				<a class="page-link" href="#">3</a>
-			</li>
-			<li class="page-item">
-				<a class="page-link" href="#">4</a>
-			</li>
-			<li class="page-item">
-				<a class="page-link" href="#">5</a>
-			</li>
-			<!-- Next -->
-			<li class="page-item">
-				<a class="page-link" href="#">&rsaquo;</a>
-			</li>
-			<!-- Next -->
-		</ul>
-	</nav>
-	<!-- END PAGINATION -->
+	<!-- Pagination -->
+	{{ $products->links('vendor.pagination.bootstrap-4') }}
```

Comment:
ouf, view jsut for the pagination, very nice 👌

Reply:
super easy to implement!